aperturedb/NotebookHelpers.py

```diff
@@ -53,12 +53,29 @@ def check_folder(folder):
 
 
 def display(images_array, save=False):
+    """
+    Show images with matplotlib
+    :::note
+    This method was used by ApertureDB to display images, but it is not recommended anymore.
+    It will not draw annotation.
+
+    Instead, when using with JSON queries, make a instance of the Images class and call the display method.
+    :
+
+    ```python
+    from aperturedb.Images import Images
+    result, response, blobs = execute_batch([{"FindImage":{"uniqueids": True}}], [], db)
+    wrapper = Images(db, response=response[0]["FindImage"]["entities"])
```

Comment:
ok, I got it - so that is what response is for.

But using this call won't display annotations either no?

Reply:
Images is pretty lazy loaded. Only if a subsequent calls to display has show_bboxes, or show_polygons, additional queries would be made. Even blobs happen lazily.

Reply:
ok, cool, so more flexibility to pull in from a more complex query that you'd done by hand.